main.go

```diff
@@ -73,6 +75,7 @@ func main() {
 			"Enabling this will ensure there is only one active controller manager.")
 	opts := zap.Options{
 		Development: true,
+		TimeEncoder: zapcore.ISO8601TimeEncoder,
```

Comment:
Fine by me. But for consistency I'm thinking we'll want all of our operators to do similar. So maybe we land this... but eventually move a common function into lib-common